libs/sdk-backend-spi/src/workspace/attributes/elements/index.ts

```diff
@@ -41,19 +41,14 @@ export interface IElementsQueryOptions {
     includeTotalCountWithoutFilters?: boolean;
 
     /**
-     * TODO what is this doing?
+     * @deprecated use withAttributeFilters or withMeasures
```

Comment:
i say let's just delete these.. i mean.. they did not work anyway so them being here is a bug :)

Reply:
I agree, I was just scared 😅

Reply:
Removed in an extra commit